TriangleApp/app/src/main/java/triangle/triangleapp/helpers/InitializationHelper.java

```diff
@@ -0,0 +1,104 @@
+package triangle.triangleapp.helpers;
+
+import android.content.Context;
+import android.util.Log;
+import triangle.triangleapp.BuildConfig;
+
+import triangle.triangleapp.ConfigHelper;
+
+
+/**
+ * Created by D2110175 on 16-6-2017.
+ */
+
+public class InitializationHelper {
+    private static int VERSION_CODE;
```

Comment:
calling getter and placing it in private variable > calling method seperate 12 times